winrt.lua

```diff
@@ -50,14 +50,27 @@
 
 	p.override(p.vstudio.vc2010, "emitFiles", function(base, prj, group, tag, fileFunc, fileCfgFunc, checkFunc)
 		if isWinRT(prj.system) then
-			if tag == "ClCompile" then
-				fileCfgFunc = table.join(fileCfgFunc or {}, {
-					m.compileAsWinRT,
-				})
-			elseif tag == "None" then
-				fileCfgFunc = table.join(fileCfgFunc or {}, {
-					m.deploymentContent,
-				})
+			local oldFn = fileCfgFunc
+			fileCfgFunc = function (...)
+				local elements
+
+				if type(oldFn) == "function" then
```

Comment:
Don't really understand this reasons behind block, oldFn might be a table instead of a function?